test/module/irohad/ametsuchi/postgres_options_test.cpp

```diff
@@ -120,9 +121,8 @@ TEST(PostgresOptionsTest, DBnameParamExist) {
               "1991",
               "petya",
               "friend",
-              dbname,  // TODO 2019.06.26 mboldyrev IR-556 change dbname
```

Comment:
IR-556 is not resolved in this PR, or in #900 . Why remove the TODO?

Reply:
Because it was not implemented during 2 years. This is old and for sure unnessesary. 

Anyway the issue is solved maintenance database name could be set from config in two ways: 1)connection string 2)connection json struct.

Reply:
See PostgresOptions